sympy/solvers/bivariate.py

```diff
@@ -6,13 +6,19 @@
 from sympy.core.power import Pow
 from sympy.core.singleton import S
 from sympy.core.symbol import Dummy
+from sympy.functions import sin, cos, sec, csc, tan, cot, Abs
 from sympy.functions.elementary.exponential import (LambertW, exp, log)
 from sympy.functions.elementary.miscellaneous import root
-from sympy.polys.polytools import Poly, factor
+from sympy.polys.polytools import Poly, factor, poly_from_expr
+from sympy.polys.polyerrors import PolificationFailed, ComputationFailed
 from sympy.core.function import _mexpand
 from sympy.simplify.simplify import separatevars
 from sympy.simplify.radsimp import collect
 from sympy.solvers.solvers import solve, _invert
+from sympy.solvers.solveset import _term_factors as tf
```

Comment:
`tf` -> `term_factors` for better readability (in the routine).

Reply:
Done.